proto/ibc/core/channel/v1/tx.proto

```diff
@@ -69,8 +69,8 @@ service Msg {
   // UpdateChannelParams defines a rpc handler method for MsgUpdateParams.
   rpc UpdateChannelParams(MsgUpdateParams) returns (MsgUpdateParamsResponse);
 
-  // PruneAcknowledgements defines a rpc handler method for MsgPruneAcknowledgements.
-  rpc PruneAcknowledgements(MsgPruneAcknowledgements) returns (MsgPruneAcknowledgementsResponse);
+  // PruneStalePacketState defines a rpc handler method for MsgPruneStalePacketState.
+  rpc PruneStalePacketState(MsgPruneStalePacketState) returns (MsgPruneStalePacketStateResponse);
```

Comment:
My vote would be for `PruneAcknowledgements` based on @AdityaSripal's comment about receipts being kind of like a subset of acks used within ibc core. Or if not going with `PruneAcknowledgements` I would drop the word "stale" as its implicit that anything that can be pruned must be stale or not required anymore.

Reply:
`PruneAcknowledgements` was fine by me. I'd also propose `PrunePacketReceiveState`. My only gripe with `PruneStalePacketState` is that it doesn't indicate what type of packet state we are pruning

Reply:
I will do then now the renaming to `PruneAcknowledgements` and MsgPruneAcknowledgements`.

Reply:
Re-renaming done. ✅